### PR TITLE
change node version so travis can build the api

### PR DESCRIPTION
**Pull request by wendyperalta**
🤞🏽 🤞🏽

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: node_js
 node_js:
-  - "4.5"
+  - "12"
```
Comment (scottgeye) [inline .travis.yml:3]: Should it LTS then so that the API can be LTS?

Comment (wendyperalta) [reply .travis.yml:3]: Isn't lts what's messing everything up?

Comment (wendyperalta) [reply .travis.yml:3]: I changed the portal api back to LTS i just haven't pushed it yet

Comment (scottgeye) [reply .travis.yml:3]: Cool, honestly, whatever works.

**Pull request closed without merging**